misc/python/materialize/benches/avro_ingest.py

```diff
@@ -15,12 +15,12 @@
 import time
 from typing import IO, NamedTuple, cast
 
-import docker
-import pg8000
+import docker  # type: ignore
+import pg8000  # type: ignore
```

Comment:
We maintain our own stubs in ./python/stubs/pg8000, including `./python/stubs/pg8000/dbapi.pyi` so I wonder why this warning appears at all.

Reply:
Good to know, I will check if I can get pyright to pick them up.

Reply:
Thanks for this info! I had to adjust the pyright configuration by specifying the path to the stubs. It can find them now. 😀

Reply:
✅